### PR TITLE
add checkmark if passed check

### PR DESCRIPTION
**Pull request by Lebron1992**

### CODE REVIEW

File: HCTextField/Source/HCTextField.swift [hunk 1]
```diff
@@ -163,6 +170,38 @@ open class HCTextField: UITextField {
...
+        }
+
+        let dividerWidth: CGFloat = 5
+        let imageViewWH = frame.size.height * 0.8
+        let containerViewWidth = imageViewWH + dividerWidth
+
+        let rightContainerView = UIView(frame: CGRect(x: 0, y: 0, width: containerViewWidth, height: imageViewWH))
+
+        let imageView = UIImageView(frame: CGRect(x: 0, y: 0, width: imageViewWH, height: imageViewWH))
+        imageView.image = UIImage(named: "checkmark")
```
Comment (laeroah) [inline HCTextField/Source/HCTextField.swift]: let the application pass the checkmark image to you. Add a optional property of checkmark.

File: HCTextField/Source/HCImages.xcassets/Contents.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (laeroah) [inline HCTextField/Source/HCImages.xcassets/Contents.json]: Don't make the image part of the textfield source code. Let the application pass the checkmark image to you. Add a optional property of checkmark. The application can choose which checkmark image to use.

**Review by laeroah — CHANGES_REQUESTED** [review]
let the application pass the checkmark image to you. Add a optional property of checkmark.

**Review by laeroah — APPROVED** [review]
:shipit:

**Comment by Lebron1992** [comment]
Good advice. Thanks!

**Pull request merged**